UPGRADING-3.x.md

```diff
@@ -37,24 +37,24 @@ e.g.
 -      {{#tooltip-on-element
 -        class="user-banner__photo__tooltip js-user-photo-tooltip"
 -        enableLazyRendering=true
--        side='right'}}
+-        side="right"}}
 +      {{#ember-tooltip
 +        class="js-user-photo-tooltip"
-+        tooltipClassName="ember-tooltip user-banner__photo__tooltip"
-+        side='right'}}
++        tooltipClass="user-banner__photo__tooltip"
++        side="right"}}
          <img src={{user.photo_url}} alt="User photo" />
```

Comment:
It's cute that we have alt tags even in tests ❤️

Reply:
A11y is key! 😄